src/infer_request.cc

```diff
@@ -443,6 +458,11 @@ InferenceRequest::RespondIfError(
           std::move(response), TRITONSERVER_RESPONSE_COMPLETE_FINAL, status),
       (request->LogRequest() + "failed to send error response").c_str());
 
+  request->ReportErrorStatistics(
+      request->model_raw_->MetricReporter().get(),
+      reason  // reason: The specific reason for the failure
```

Comment:
```suggestion
      reason
```

Reply:
please remove the comment, it's explained in function docstring and variable name